javascript-typing-tutor/main.js

```diff
@@ -0,0 +1,56 @@
+const $letter = document.querySelectorAll('span');
+const $message = document.querySelector('.message');
+const $resetButton = document.querySelector('.reset-button');
+
+let i = 0;
+let typeCount = 0;
+
+console.log($letter.length)
```

Comment:
Remove all console logs before submitting! They are only for testing / debugging purposes, unless we specifically ask for them!